core/dbt/include/global_project/macros/materializations/models/incremental/strategies.sql

```diff
@@ -0,0 +1,83 @@
+{% macro get_incremental_append_sql(arg_dict) %}
+
+  {{ return(adapter.dispatch('get_incremental_append_sql', 'dbt')(arg_dict)) }}
+
+{% endmacro %}
+
+
+{% macro default__get_incremental_append_sql(arg_dict) %}
+
+  {% do return(get_insert_into_sql(arg_dict["target_relation"], arg_dict["temp_relation"], arg_dict["dest_columns"])) %}
```

Comment:
I didn't get this at first, but makes sense that we need to "unpack" the arguments to preserve the existing signature of the `get_insert_into_sql` macro (for backwards compatibility, for anyone who's overridden/reimplemented it).

Reply:
Yes. I am thinking of switching to using keyword arguments instead of positional though.